text/metrics/0000-otlp-export-behavior.md

```diff
@@ -0,0 +1,49 @@
+# OTLP Configurable Export Behavior
+ Add OTLP exporters support for configurable export behavior.
+
+ The export behaviors are exporting cumulative values since start time by default, and exporting delta values per collection interval when configured. 
+
+## Motivation 
+
+1. Metric backends such as Prometheus and Prometheus Remote Write integrated storages requires cumulative values for cumulative/adding metrics rather than delta value per collection interval. Thus, to export metrics generated by the SDK using the collector, incoming values from the SDK should be in cumulative form. However, backends like Statsd expects delta value of each collection interval. To support different backends, OTLP metric export behavior should be configurable, with cumulative export as default. The metrics SIG meeting concluded that cumulative export behavior is more reliable, as explained in the following section.  
```

Comment:
Maybe say "backends that make use of the Prometheus Remote Write API"

Reply:
# OTLP Configurable Export Behavior
 Add support for configurable behavior in OTLP exporters